docs/community/release-notes.md

```diff
@@ -42,11 +42,52 @@ You can determine your currently installed version using `pip show`:
 
 ### 3.9.0
 
-**Date**: Unreleased
+**Date**: [1st October 2018][3.9.0-milestone]
 
+* Improvements to ViewSet extra actions [#5605][gh5605]
+* Fix `action` support for ViewSet suffixes [#6081][gh6081]
```

Comment:
I might remove this from the changelog, as this fixes an unreleased bug from 5605 above it.